chore: add UpdateRequests reconciliation details (#1066)
kyverno/website: content/en/docs/Writing policies/generate.md

* `Pending`: the request is yet to be processed or the resource has not been created
* `Skip`: marked when triggering the generate policy by adding a label/annotation to the existing resource, while the selector is not defined in the policy itself.

> Note that Kyverno retries up to three times to reconcile a `Failed` `UpdateRequest`, and the `UpdateRequest` will be garbage collected if it exceeds the retry threshold.

Reviewer: ```suggestion
Note that Kyverno will retry up to three times to reconcile an `UpdateRequest` in a `Failed` status. The `UpdateRequest` will be garbage collected if it exceeds the retry threshold.
```
